spec/models/recipe_spec.rb

```diff
@@ -0,0 +1,62 @@
+# /spec/models/recipe_spec.rb
+require 'rails_helper'
+
+RSpec.describe Recipe, type: :model do
+  let(:recipe) { Recipe.new } # Define a recipe object
+
+  it 'is not valid without a name' do
+    recipe.name = nil
+    recipe.valid?
+    expect(recipe).not_to be_valid
+    expect(recipe.errors[:name]).to include("can't be blank")
+  end
+
+  it 'is not valid with a name less than 3 characters' do
+    recipe.name = 'ab'
+    recipe.valid?
+    expect(recipe).not_to be_valid
+    expect(recipe.errors[:name]).to include('is too short (minimum is 3 characters)')
+  end
+
+  it 'is not valid with a name more than 50 characters' do
+    recipe.name = 'a' * 51
+    recipe.valid?
+    expect(recipe).not_to be_valid
+    expect(recipe.errors[:name]).to include('is too long (maximum is 50 characters)')
+  end
+
+  it 'is not valid without a description' do
+    recipe.description = nil
```

Comment:
- [ ] Nice job improving on the test cases, however, 4 test cases are still failing. Let's ensure all test cases are passing!

![image](https://github.com/AbdelazizSoliman/Recipe_app/assets/12745474/5734bf1b-2f2d-4311-b003-19c6ad27e9b9)